public/API/API_GetClaims.php

```diff
@@ -1,14 +1,13 @@
 <?php
 
-use App\Community\Enums\ClaimFilters;
-use Illuminate\Support\Facades\Validator;
-use Illuminate\Validation\Rule;
-
 /*
  *  API_GetClaims - returns information about 1000 max set claims, sorted by latest `Created` date.
+ *    o : offset - number of entries to skip (default: 0)
+ *    c : count - number of entries to return (default: 100, max: 500)
  *    k : claim kind - 1 for completed, 2 for dropped, 3 for expired (default: 1)
- *
- *  array
+ *  int        Count                number of claims returned in the response
+ *  int        Total                number of claims that actually exist
+ *  array      Results
```

Comment:
I realize this is necessary to return the additional fields, but it's a breaking change. Make sure it's well communicated in the release notes (possibly even beforehand).